src/main/java/javaemul/internal/ArrayHelper.java

```diff
@@ -73,6 +73,10 @@ public static void insertTo(Object array, int index, Object[] values) {
 		copy(values, 0, array, index, values.length, false);
 	}
 
+	public static void insertToArr(Object array, int index, Object[] values) {
```

Comment:
It would be better if you could rename it to something more meaningful and add a Javadoc explaining why we need an overload.
Thank you very much

Reply:
I created a commit for that. It is good enough? :) 
https://github.com/j4ts/j4ts/commit/d89636a8d99238cb7c87936cdf25436fb62d23ed